aggsender/aggsender.go

```diff
@@ -330,12 +326,6 @@ func (a *AggSender) sendCertificate(ctx context.Context) (*agglayertypes.Certifi
 		a.log.Warn("dry run mode enabled, skipping sending certificate")
 		return certificate, nil
 	}
-	if rateLimitSleepTime := a.rateLimiter.Call("sendCertificate", false); rateLimitSleepTime != nil {
```

Comment:
@joanestebanr were we actually not limiting the user? do we want to keep the same behaviour as well in current calls?